pkg/pss/evaluate.go

```diff
@@ -84,36 +84,46 @@ func exemptKyvernoExclusion(defaultCheckResults, excludeCheckResults []pssutils.
 	return newDefaultCheckResults
 }
 
-func parseVersion(rule *kyvernov1.PodSecurity) (*api.LevelVersion, error) {
+func ParseVersion(level api.Level, version string) (*api.LevelVersion, error) {
```

Comment:
@itsgurmannatsohal @YTGhost  - please be aware of this change, it might result in conflicts.

Reply:
Noted